packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -265,6 +265,12 @@ class _TooltipOverlay extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    ThemeData theme = Theme.of(context);
+    ThemeData darkTheme = new ThemeData(
+      brightness: Brightness.dark,
+      accentColor: theme.accentColor,
```

Comment:
why do we care about the accent color? Does it affect the backgroundColor?

Also, since you don't propagate the typography across from the original theme here, you're not actually going to be solving your problem.